src/briefcase/commands/create.py

```diff
@@ -633,6 +633,7 @@ def _install_app_requirements(
         app_packages_path: Path,
         progress_message: str = "Installing app requirements...",
         pip_args: list[str] | None = None,
+        install_hint: str = "",
         pip_kwargs: dict[str, str] | None = None,
```

Comment:
From an organisational perspective, it would be preferable to keep `pip_args` and `pip_kwargs` together.
```suggestion
        pip_args: list[str] | None = None,
        pip_kwargs: dict[str, str] | None = None,
        install_hint: str = "",
```